### ISSUE TITLE
change ctrl+alt+t key binding to T

### ISSUE THREAD

**Issue by thiswillbeyourgithub** [burtonator/polar-bookshelf#995]
**Is your feature request related to a problem? Please describe.**
When reading a document, the keybinding "CTRL + ALT + T" doesn't seem to work on my linux because the default behavior on ubuntu it to create a new terminal window.

**Describe the solution you'd like**
I think it would be good to change this shortcut to simply "T". It would make it easier to use repeatedly. Also maybe T+1 could be yellow, T+2 another color etc ?

**Describe alternatives you've considered**
It doesn't seem reasonable to disable the default shortcut of my ubuntu to allow the polar keybinding to work. And it seems suboptimal to me to have to typep on 3 keys to highlight text if I have to frequently do it.

**Additional context**
I am on ubuntu 18.04

Congrats on the work so far, loving every steps of your project :)

ps :
* this page contains the word "pgoress" instead of "progress" : https://getpolarized.io/incremental-reading.html

* this page contains a 404 link about epub format https://getpolarized.io/docs/frequently-asked-questions.html

**Comment by burtonator** [comment]
Agreed. I think I'm going to change all the main bindings to not use modifiers and just be 't' based on the context.  

One thing I need  to think about is how to handle it when typing in a form box but I think it will work just fine.

**Comment by Rumata888** [comment]
Maybe it would be better to make it Ctrl + <Letter>. After all it's the agreed way. Just ctrl plus letter is used in all applications, so it's rarely hooked by the systems and it doesn't intersect with typing (unless maybe Ctrl+Shift for language change on Windows)

**Comment by thiswillbeyourgithub** [comment]
I understand this "compatibility argument" but if you are highlighting text from a pdf every 20 seconds for a whole hour your left thumb is going to thank me. **Holding down** a key while pressing another is more intensive on the hand than *just pressing* on one key.

If you this app is intended for work it's not negligible in my opinion

**Comment by Rumata888** [comment]
You may have a point. Maybe use not T, but R,G,Y. They are close and obviously convey the meaning well.

**Comment by thiswillbeyourgithub** [comment]
I disagree :)

In my opinion, Polar is developping fast and will add lots of feature. Adding R/G/Y for coloring in a specific color is underestimating the number of features to come as well as forgetting the other colors. In the annotation bar you can select way more colors like shades of green etc.

Are you a Anki user ? It reminds me of the card editor in anki, their editing is so rich that they have "sequence keybinding" (don't know how it's called), for example if you want to invoke latex equation you type ctrl+T **then** E (and yes, it's quite a lot for the wrist but understandable to me given the number of shortcuts, the objective is not to end up like vim !)

To sum it up (and tell me if I'm not being clear) I think a sequence keybinding is the way.

My take : 
* T+number = a color
* T twice = reuse last color

Efficiency of keybindings (again, without having to end up like vim) is key for all those thesis guys.

What is your opinion on this @burtonator ?

**Comment by Rumata888** [comment]
Maybe. T twice, however can become really frustrating if you use one color. It's a sequential command.

**Comment by thiswillbeyourgithub** [comment]
Good point. 

My second take then : 

* T = color the text using the last used color
* pressing then a number changes the applied color, in effect it's the same as T+number=specific color

It might be a bit ressource intensive but it's seems decently intuitive. What do you guys think ?

**Comment by Rumata888** [comment]
Definitely better ergonomically

**Comment by thiswillbeyourgithub** [comment]
No update on this issue ? I think it would speed up annotating by a huge margin
